tests/test_visitors/test_ast/test_annotations/test_union_annotations.py

```diff
@@ -0,0 +1,37 @@
+import pytest
+
+from wemake_python_styleguide.violations.best_practices import (
+    DisallowUnionTypeViolation,
+)
+from wemake_python_styleguide.visitors.ast.annotations import (
+    WrongAnnotationVisitor,
+)
+
+functions_breaking_rule = [
```

Comment:
This can be use directly in `@pytest.mark.parametrize`

Reply:
Done :heavy_check_mark: